script/build.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Single source of truth for building Firecloud-Orchestration.
+# @ Jackie Roberti
+#
+# Provide command line options to do one or several things:
+#   jar : build orch jar
+#   -d | --docker : provide arg either "build" or "push", to build and push docker image
+# Jenkins build job should run with all options, for example,
+#   ./docker/build.sh jar -d push
```

Comment:
script dir, not docker

Or you could move this to the docker dir?  Shrug, whatever makes sense.